### PR TITLE
VEN-1225 | Add translations for Finnish and Swedish

### PR DESCRIPTION
**Pull request by klemmari1**
## Description :sparkles:

Ensure that email subjects have translations. Removed / fixed all fuzzy translations so that at least there are no incorrect translations in the system.

### Closes :no_good_woman:
**[VEN-1225](https://helsinkisolutionoffice.atlassian.net/browse/VEN-1225)**

### CODE REVIEW

File: locale/fi/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -258,26 +256,20 @@ msgstr "trailerin rekisterinumero"
...
+msgstr "Venepaikkahakemus hylätty"
 
 msgid "Winter storage application created"
 msgstr "Talvisäilytyshakemus luotu"
 
-#, fuzzy
-#| msgid "Winter storage application created"
 msgid "Unmarked winter storage application created"
-msgstr "Talvisäilytyshakemus luotu"
+msgstr "Merkitsemätön talvisäilytyshakemus luotu"
```
Comment (TatuArvela) [inline locale/fi/LC_MESSAGES/django.po:272]: Nostojärjestysilmoitus luotu

File: locale/fi/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -627,36 +647,26 @@ msgstr "Valittu venepaikka ei ole aktiivinen"
...
-#| msgid "order number"
 msgid "sticker number"
-msgstr "tilausnumero"
+msgstr "tarran numero"
 
-#, fuzzy
-#| msgid "order number"
 msgid "sticker posted"
-msgstr "tilausnumero"
+msgstr ""
```
Comment (TatuArvela) [inline locale/fi/LC_MESSAGES/django.po:657]: tarra postitettu

File: locale/fi/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -627,36 +647,26 @@ msgstr "Valittu venepaikka ei ole aktiivinen"
...
 msgstr "talvisäilytyspaikan vuokrasopimus"
 
 msgid "winter storage leases"
 msgstr "talvisäilytyspaikkojen vuokrasopimukset"
 
-#, fuzzy
-#| msgid "Lease cannot have both place and area assigned"
 msgid "Lease cannot have both place and section assigned"
-msgstr "Vuokrasopimukseen ei voida liittää sekä paikkaa että aluetta"
+msgstr "Vuokrasopimukseen ei voida liittää sekä paikkaa että osa-aluetta"
```
Comment (TatuArvela) [inline locale/fi/LC_MESSAGES/django.po:666]: Not 100% sure if this is the term for "section"

File: locale/fi/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -807,13 +841,17 @@ msgstr "Prosenttimäärä"
...
 msgstr "Hylätty"
 
-#, fuzzy
-#| msgid "order"
+msgid "Accepted"
+msgstr "Hyväksytty"
+
 msgid "Lease order"
-msgstr "tilaus"
+msgstr "Vuokratilaus"
```
Comment (TatuArvela) [inline locale/fi/LC_MESSAGES/django.po:854]: Not sure about this one, but I don't know what else it should be

Comment (TatuArvela) [reply locale/fi/LC_MESSAGES/django.po:854]: We don't show the word "tilaus" to customers though

File: locale/fi/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -824,20 +862,14 @@ msgstr ""
 msgid "Renew berth order"
 msgstr ""
 
-#, fuzzy
-#| msgid "berth switch reason"
 msgid "Berth switch order"
-msgstr "paikanvaihdon syy"
+msgstr "paikanvaihdotilaus"
```
Comment (TatuArvela) [inline locale/fi/LC_MESSAGES/django.po:866]: Venepaikan vaihto

File: locale/fi/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -996,39 +1060,56 @@ msgstr "Toisella tilauksella on jo sama tunnus"
...
+msgstr ""
 
 msgid "The order is not valid anymore"
 msgstr "Tilaus ei ole enää voimassa"
 
-#, fuzzy
-#| msgid "winter storage area"
 msgid "Cannot cancel Unmarked winter storage order"
-msgstr "talvisäilytysalue"
+msgstr "Merkitsemätöntä talvisäilytystilausta ei voi peruuttaa"
```
Comment (TatuArvela) [inline locale/fi/LC_MESSAGES/django.po:1082]: Nostojärjestyspaikkaa ei voi peruuttaa

File: locale/fi/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -1179,6 +1260,9 @@ msgstr "numero"
 msgid "is accessible"
 msgstr "on esteetön"
 
+msgid "is invoiceable"
+msgstr "On laskutettavissa"
```
Comment (TatuArvela) [inline locale/fi/LC_MESSAGES/django.po:1264]: Lowercase maybe? Not sure where this is used

File: locale/sv/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -288,6 +280,33 @@ msgstr "Svenska"
...
+msgstr "Signerad"
+
+msgid "Deleted"
+msgstr "Raderad"
+
+msgid "Cancelled"
+msgstr "Anullerad"
+
+msgid "contract status"
+msgstr "kontrakt status"
```
Comment (TatuArvela) [inline locale/sv/LC_MESSAGES/django.po:296]: We use "avtal" for contract in some other places

File: locale/sv/LC_MESSAGES/django.po [hunk 9]
```diff
@@ -700,47 +706,82 @@ msgid "winter storage lease changes"
 msgstr "ändringar i hyresavtal för vinteruppläggningsplats"
 
 msgid "Next season invoices sent"
-msgstr ""
+msgstr "Nästa säsong fakturor skickat"
```
Comment (TatuArvela) [inline locale/sv/LC_MESSAGES/django.po:709]: Nästa säsongs fakturor har skickats

File: locale/sv/LC_MESSAGES/django.po [hunk 10]
```diff
@@ -818,36 +861,54 @@ msgstr ""
...
-#| msgid "berth switch reason"
 msgid "Berth switch order"
-msgstr "orsak till byte av båtplats"
+msgstr "ordning till byte av båtplats"
 
-#, fuzzy
-#| msgid "winter storage area"
 msgid "Winter storage order"
-msgstr "vinterupplagsområde"
+msgstr "vinteruppläggningsordning"
```
Comment (TatuArvela) [inline locale/sv/LC_MESSAGES/django.po:868]: Check if the case has to match

File: locale/sv/LC_MESSAGES/django.po [hunk 11]
```diff
@@ -930,19 +992,41 @@ msgid "Cannot change the product assigned to this order line"
...
 msgstr "poster i beställningshistorik"
 
 msgid "token"
 msgstr ""
 
-#, fuzzy
-#| msgid "Cancelled"
 msgid "cancelled"
-msgstr "Anullerad"
+msgstr "anullerad"
```
Comment (TatuArvela) [inline locale/sv/LC_MESSAGES/django.po:1005]: annullerad

**Review by TatuArvela — COMMENTED** [review]
Some are still missing, but is there a need to translate all of these? Added some comments

**Comment by klemmari1** [comment]
Thanks for the suggestions! Not everything needs to be translated I think because part of the translations are for error messages which are not shown for the customers / admins. I focused mostly on the email related translations.

File: locale/fi/LC_MESSAGES/django.po (hunk 4 above)
Comment (haavikko) [reply locale/fi/LC_MESSAGES/django.po:854]: Should it be "Vuokralasku"?

Comment (haavikko) [reply locale/fi/LC_MESSAGES/django.po:854]: Or "Venepaikan lasku"?

File: locale/fi/LC_MESSAGES/django.po [hunk 12]
```diff
@@ -894,10 +929,8 @@ msgstr "Kiinteät palvelut ovat voimassa vain kaudella"
 msgid "order number"
 msgstr "tilausnumero"
 
-#, fuzzy
-#| msgid "berth type"
 msgid "order type"
-msgstr "venepaikan tyyppi"
+msgstr "tilauksen tyyppi"
```
Comment (haavikko) [inline locale/fi/LC_MESSAGES/django.po:933]: Laskun tyyppi?

File: locale/fi/LC_MESSAGES/django.po [hunk 13]
```diff
@@ -931,6 +964,9 @@ msgstr "Tähän tilaukseen liitettyä tuotetta ei voida vaihtaa"
 msgid "Cannot change the lease associated with this order"
 msgstr "Tähän tilaukseen liitettyä vuokrasopimusta ei voida vaihtaa"
 
+msgid "Cannot change due date of this order"
+msgstr "Tilauksen eräpäivää ei voida vaihtaa"
+
```
Comment (haavikko) [inline locale/fi/LC_MESSAGES/django.po:969]: Laskun eräpäivää ei voida vaihtaa

File: locale/fi/LC_MESSAGES/django.po [hunk 14]
```diff
@@ -949,19 +985,41 @@ msgstr "Tähän tilausriviin liitettyä tilausta ei voida vaihtaa"
 msgid "Cannot change the product assigned to this order line"
 msgstr "Tähän tilausriviin määritettyä tuotetta ei voida vaihtaa"
 
-msgid "order log entry"
-msgstr "tilauslokikirjaus"
+msgid "Lease must have a paid order"
+msgstr "Vuokrasopimuksella tulee olla maksettu tilaus"
```
Comment (haavikko) [inline locale/fi/LC_MESSAGES/django.po:989]: Vuokrasopimuksella tulee olla maksettu lasku

File: locale/fi/LC_MESSAGES/django.po [hunk 15]
```diff
@@ -949,19 +985,41 @@ msgstr "Tähän tilausriviin liitettyä tilausta ei voida vaihtaa"
...
+msgstr "peruutettu"
+
+msgid "refund id"
+msgstr "palautuksen tunniste"
+
+msgid "amount"
+msgstr "summa"
+
+msgid "Cannot refund orders that are not paid"
+msgstr ""
```
Comment (haavikko) [inline locale/fi/LC_MESSAGES/django.po:1007]: Maksamattomia laskuja ei voi hyvittää

File: locale/fi/LC_MESSAGES/django.po (hunk 3 above)
Comment (klemmari1) [reply locale/fi/LC_MESSAGES/django.po:666]: In other places where "section" was used, it was translated as "osa-alue". Not sure if it's correct but it's consistent :grin:

File: locale/fi/LC_MESSAGES/django.po (hunk 12 above)
Comment (klemmari1) [reply locale/fi/LC_MESSAGES/django.po:933]: Many existing translations translate "order" as "tilaus". Should we change them all? :thinking:

File: locale/fi/LC_MESSAGES/django.po [hunk 16]
```diff
@@ -996,39 +1060,56 @@ msgstr "Toisella tilauksella on jo sama tunnus"
...
 msgstr "Lisäpalvelu on jo olemassa"
 
 msgid "Lease with the given ID does not exist"
 msgstr "Annetulla tunnuksella ei ole vuokrasopimusta"
 
-#, fuzzy
-#| msgid "Storage on ice"
 msgid "Only storage on ice supported"
-msgstr "Jäihin jättö"
+msgstr ""
```
Comment (haavikko) [inline locale/fi/LC_MESSAGES/django.po:1073]: Vain jäissäsäilytys tuettu

**Review by haavikko — APPROVED** [review]
I think/hope this fixes the most important issues with translations

**Comment by a-rmz** [comment]
Otherwise, we'll hear about it 😅 let's do this!

**Pull request merged**